openbb_platform/openbb/package/__extensions__.py

```diff
@@ -11,13 +11,16 @@ class Extensions(Container):
     /crypto
     /currency
     /derivatives
+    /econometrics
```

Comment:
We need to keep the community extensions out of the static assets - extensions and providers.